diff --git dom/media/eme/MediaKeySystemAccess.cpp dom/media/eme/MediaKeySystemAccess.cpp
index 6d596cf..a54552d 100644
--- dom/media/eme/MediaKeySystemAccess.cpp
+++ dom/media/eme/MediaKeySystemAccess.cpp
@@ -99,10 +99,14 @@ MediaKeySystemAccess::IsKeySystemSupported(const nsAString& aKeySystem)
     return false;
   }
 
-  if (aKeySystem.EqualsLiteral("org.w3.clearkey") &&
-      HaveGMPFor(mps,
-                 NS_LITERAL_CSTRING("org.w3.clearkey"),
-                 NS_LITERAL_CSTRING("eme-decrypt"))) {
+  if ((aKeySystem.EqualsLiteral("org.w3.clearkey") &&
+       HaveGMPFor(mps,
+                  NS_LITERAL_CSTRING("org.w3.clearkey"),
+                  NS_LITERAL_CSTRING("eme-decrypt")))
+      || (aKeySystem.EqualsLiteral("com.marlin-drm.broadband.1_0") &&
+          HaveGMPFor(mps,
+                     NS_LITERAL_CSTRING("com.marlin-drm.broadband.1_0"),
+                     NS_LITERAL_CSTRING("eme-decrypt")))) {
     return true;
   }
 
